frontstage/controllers/collection_exercise_controller.py

```diff
@@ -53,10 +53,13 @@ def get_collection_exercise_events(collection_exercise_id):
     return response.json()
 
 
-def get_collection_exercises_for_survey(survey_id):
+def get_collection_exercises_for_survey(survey_id, live_only=None):
     logger.debug('Retrieving collection exercises for survey', survey_id=survey_id)
 
-    url = f"{app.config['COLLECTION_EXERCISE_URL']}/collectionexercises/survey/{survey_id}"
+    if live_only is True:
```

Comment:
can just say `if live_only:`

Reply:
He wants to make sure it only goes to that one if the parameter passed is True as if you did that it would use it if anything was passed.